docs/github-actions.md

```diff
@@ -0,0 +1,313 @@
+---
+id: github-actions
+title: Testrunner Toolkit with GitHub Actions
+sidebar_label: GitHub Actions
+---
+
+The examples on this page can be applied to virtually any deployment provided that you already have some existing automated tests, and are either the maintainer or an admin of the target repository. 
+
+Your permissions must include:
+
+* ability to create and manage workflows
+* ability to create and store [GitHub secrets](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)
+
+## What You'll Need
+
+* [GitHub Account](https://github.com/join)
+* [Sauce Labs Account](https://saucelabs.com/sign-up)
+
+## Create GitHub Secrets
+
+The first order of business is to export your [Sauce Labs account credentials]() and store them as GitHub Secrets.
+
+* Navigate to your project repository and select the __settings__ icon
+    
+    ![GitHub Settings](assets/github-settings.png)
+
+* Select __Secrets__
+* Click the __New secret__ button
+* Add the following:
+    * Name: `SAUCE_USERNAME`
+    * Value: 'your-sauce-username'
+* Click __Add secret__ to finish.
+* Repeat the same steps above for your `SAUCE_ACCESS_KEY`
+
+## Create the `saucectl` Configuration
+
+Create the `.sauce` directory at the root of your project and add a `config.yaml` file that points to your existing `tests` directory. 
+
+With the `suites` field you can specify a group of tests as well as the browser `settings` you wish to use.
+
+Below are some examples:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```sh
+# ./.sauce/puppeteer.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Puppeteer Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
```

Comment:
`browserVersion` is currently not officially supported, since the framework images are paired with specific image versions. So you can select different browsers, not but different versions.